code401-challenges/src/main/java/code401/challenges/linkedlist/LinkedList.java

```diff
@@ -0,0 +1,66 @@
+package code401.challenges.linkedlist;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class LinkedList {
+    Node head;
+    int size;
+
+    LinkedList() {
+        //upon instantiation the head will point to null
+        this.head = null;
+        size = 0;
+    }
+
+    //function to insert a new node into the front of the linked list
+    public void insert(int value) {
+        try{
+            Node current = new Node();
+            current.value = value;
+            if (this.head == null) {
```

Comment:
same comment as above, you use `this.head` once and then use `head` everywhere else. Try to remain consistent